budget_control_transfer_dimension/models/budget_transfer_item.py

```diff
@@ -55,7 +159,9 @@ def _compute_source_analytic_tag_all(self):
                 )
             ):
                 continue
-            doc.source_analytic_tag_ids = analytic_tag_ids
+            doc.source_analytic_tag_ids = (
+                len(analytic_tag_ids) == 1 and analytic_tag_ids or False
+            )
```

Comment:
```suggestion
            doc.source_analytic_tag_ids = (
                len(analytic_tag_ids) == len(dimension_fields) and analytic_tag_ids or False
            )
```

Reply:
The dimensions must more than 1.

Reply:
user should select 1 only.
if analytic_tag_ids has 1 line, This function will help user default dimension.
if analytic_tag_ids more than 1 user must select field manual.